migrate/20250110_add_kubernetes_models.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:kubernetes_cluster) do
+      column :id, :uuid, primary_key: true, default: Sequel.lit("gen_random_uuid()")
+      column :name, :text, null: false
+      column :cp_node_count, :integer, null: false
+      column :kubernetes_version, :text, null: false
+      column :location, :text, collate: '"C"', null: false
+      column :created_at, :timestamp, null: false, default: Sequel::CURRENT_TIMESTAMP
+
+      check { (cp_node_count =~ [1, 3]) }
+      foreign_key :private_subnet_id, :private_subnet, type: :uuid, null: false
+      foreign_key :api_server_lb_id, :load_balancer, type: :uuid, null: true
+    end
+
+    create_table(:kubernetes_clusters_vms) do
+      primary_key :id, :uuid, default: Sequel.lit("gen_random_uuid()")
```

Comment:
Do we need the `id` column in the middle table? Isn't the `[:kubernetes_cluster_id, :vm_id]` pair enough?